WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java

```diff
@@ -265,12 +314,52 @@ private void uploadBillingDataAndVerify() throws Exception
         updateChargeRates();
     }
 
+    @LogMethod
+    @Override
+    protected void populateProjectRecords() throws Exception
+    {
+        InsertRowsCommand insertCmd = new InsertRowsCommand("ehr", "project");
+
+        Map<String,Object> rowMap = new HashMap<>();
+        rowMap.put("project", PROTOCOL_PROJECT_ID);
+        rowMap.put("name", PROTOCOL_PROJECT_ID);
+        rowMap.put("protocol", PROTOCOL_ID);
+        rowMap.put("account", ACCOUNT_ID_1);
+        rowMap.put("investigatorId", "1");
+        insertCmd.addRow(rowMap);
```

Comment:
<nit-pick>
Alternatively you could do:

insertCmd.addRow(Map.of(
"project", PROTOCOL_PROJECT_ID,
"name", PROTOCOL_PROJECT_ID,
"protocol", PROTOCOL_ID,
"account", ACCOUNT_ID_1,
"investigatorId", "1"));

Nothing wrong with the way it is, just an alternative way to do it.